_posts/2020-08-11-backports-application.adoc

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: From zero to hero on backporting pull-requests
+date: 2020-08-11
+tags: user-story
+author: gastaldi
+---
+= From zero to hero on backporting pull-requests
+
+Before we dig into the solution, let me give you a quick explanation about our release process
+
+== Release process
+=== Minor versions
```

Comment:
I would say major versions here. In Quarkus, 1.x is really a major given we push a ton of new features to it.

Reply:
I think calling it major versions would be confusing to users, since semantic versioning 
states that the format is `MAJOR.MINOR.PATCH`, unless we put a paragraph saying that we don't follow semantic versioning (well, if we really followed it since the beginning, we'd probably have been in Quarkus `3000.0.0.Final` given the amount of API changes we introduced :smiley:)